README_first.ipynb

```diff
@@ -1,60 +1,124 @@
 {
```

Comment:
<p>I would:</p><p> </p><ul><li>drop the exclamation mark(s)</li><li>Split with two sub-heading for the domain of the task (audio and NLP)</li><li>Have basically the relevant tabular entries from the Hugging Face optimum notebooks/README.md</li></ul><p> </p><p>Also double check language:</p><ul><li>"our our"</li><li>"to help you get you started"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>